diff --git a/contrib/dnsmasq/docker0.conf b/contrib/dnsmasq/docker0.conf
index d99bf88..985c92e 100644
--- a/contrib/dnsmasq/docker0.conf
+++ b/contrib/dnsmasq/docker0.conf
@@ -7,6 +7,7 @@ dhcp-host=52:54:00:a1:9c:ae,172.17.0.21,1h
 dhcp-host=52:54:00:b2:2f:86,172.17.0.22,1h
 dhcp-host=52:54:00:c3:61:77,172.17.0.23,1h
 dhcp-host=52:54:00:d7:99:c7,172.17.0.24,1h
+dhcp-host=52:54:00:d7:99:ab,172.17.0.25,1h
 
 enable-tftp
 tftp-root=/var/lib/tftpboot
@@ -25,4 +26,5 @@ address=/node1.example.com/172.17.0.21
 address=/node2.example.com/172.17.0.22
 address=/node3.example.com/172.17.0.23
 address=/node4.example.com/172.17.0.24
+address=/node5.example.com/172.17.0.25
 address=/cluster.example.com/172.17.0.21
